src/common/components/traffic-lights/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+
+import styles from './traffic-light.css';
+
+export const SIGNALS = {
+  DEFAULT: 0,
+  ACTIVE: 1,
+  DONE: 2
+};
+
+const stateStyles = [
+  false, // there is no style modifier for the default state
+  styles.active,
+  styles.done
+];
+
+export class Signal extends Component {
+
+  render() {
+    const { state, label, message } = this.props;
+    const signalStyle = stateStyles[state];
+
+    return (
+      <div className={ styles.box }>
+        <div className={ classNames(styles.signal, signalStyle) }>
+          { (state === SIGNALS.DONE) ? '\u00a0' : label }
```

Comment:
don't want to leave and empty node, else positioning of the message underneath is upset